src/styles/index.scss

```diff
@@ -1,3 +1,33 @@
+@use './utils/variables' as v;
+@use './utils/mixins' as m;
+@import './utils/stars-block';
+@import './utils/card-block';
+@import './utils/container';
+@import './utils/variables';
```

Comment:
You are using both `@use` and `@import` for the same files (`variables`). It's recommended to use only `@use` for better modularity and to avoid potential conflicts or redundancy.